### DISCUSSION TITLE
Watt App and 2 Homes ?

### DISCUSSION THREAD

**Discussion by GeorgeChSakkas** [tnmendes/watt#124] [Q&A]
On the main screen there is a graph showing the consumption. Is it possible to define 2 houses and show me the consumption for each house separately? 

I created the spaces below per house and inside I have put the devices I have in each house, but on the main page I could not define different houses.

![image](https://github.com/tnmendes/watt/assets/134599621/a588deb8-92c5-46c3-9dd7-ec2aff3bd6e9)

**Comment by tnmendes** [comment]
Hey @GeorgeChSakkas thank you for the suggestion, I will try to add an option so that in each space you can see an aggregated graph of that space

**Reply by GeorgeChSakkas** [reply]
Thanks!
